types/node/index.d.ts

```diff
@@ -376,20 +376,27 @@ declare var Buffer: {
      * The optional {byteOffset} and {length} arguments specify a memory range
      * within the {arrayBuffer} that will be shared by the Buffer.
      *
-     * @param arrayBuffer The .buffer property of a TypedArray or a new ArrayBuffer()
+     * @param arrayBuffer The .buffer property of any TypedArray or a new ArrayBuffer()
      */
     from(arrayBuffer: ArrayBuffer, byteOffset?: number, length?: number): Buffer;
+    // from(arrayBuffer: SharedArrayBuffer, byteOffset?: number, length?: number): Buffer;
     /**
      * Creates a new Buffer using the passed {data}
      * @param data data to create a new Buffer
      */
-    from(data: any[] | string | ArrayBuffer | Uint8Array /*| TypedArray*/): Buffer;
+    from(data: any[]): Buffer;
+    from(data: Uint8Array): Buffer;
     /**
      * Creates a new Buffer containing the given JavaScript string {str}.
      * If provided, the {encoding} parameter identifies the character encoding.
      * If not provided, {encoding} defaults to 'utf8'.
      */
     from(str: string, encoding?: string): Buffer;
+    /**
+     * Creates a new Buffer using the passed {data}
+     * @param values to create a new Buffer
+     */
+    of(...items: number[]): Buffer;
```

Comment:
`of` is not a member of `Buffer`, it's available in `Buffer` via base `Uint8Array` so it should not be here.

Reply:
Yeah; I just did a hotfix because a build failed because it couldn't find a definition of `Buffer.of`; so I added it.